src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19509.cs

```diff
@@ -23,6 +23,8 @@ public async Task EntryTextColorStopsWorkingAfterPropertyIsUpdatedFromBinding()
 
 			await Task.Delay(1000); // Wait for Ripple Effect animation to complete.
 
+			await Task.Delay(1000); // Wait to complete the Ripple Effect animation.
```

Comment:
do we need both here @jsuarezruiz  ? seems we have 2 delays

Reply:
Not, this looks like a merge issue. Fixed.